docker-compose.dev.yml

```diff
@@ -25,7 +25,7 @@ services:
     depends_on:
       - db
     environment:
-      - DB_HOST=db
```

Comment:
would it be helpful to make these same changes in `docker-compose.yml`?

Reply:
The intent of using `load_mongodump.sh` is to load the dataset into mongo db. 
A couple of observations:
- We are loading the config file in `load_mongodump.sh` from docker-compose.dev.yml.
`CONFIG_FILE="$SCRIPT_DIR/../../docker-compose.dev.yml"`
- We would need to make some changes in `load_mongodump.sh` if we want to configure for both `docker-compose.yml` and `docker-compose.dev.yml`.
- We can anyways load the dataset with the current changes for the public dashboard db, do you think the additional changes would be a necessity? Moreover, it's implemented as in with op-admin dashboard.

Please let me know what you think.

Reply:
My thought was that the updated default with the formatting might make it easier to switch that dataset in use when editing the docker file (ie update from `mongodb://db/DB_NAME` to `mongodb://db/openpath_prod_open_acess` instead of from `db`, but it sounds like there are more technical reasons that differ between the two files, so I'm ok with leaving the other dockerfile alone

Reply:
Just to clarify, users don't need to rename `DB_HOST=mongodb://db/DB_NAME` manually; the script edits the file directly.

Reply:
Reverting back to `DB_HOST=db` such that it's consistent between `docker-compose.dev.yml` and `docker-compose.yml`

Reply:
Similarly, I don't understand why you reverted this change. I am also not sure how it works, given that `load_mongodump.sh` expects a `mongodb://` URL so that it can change the `DB_NAME` appropriately, I bet that this was not tested, because I don't see how it could work.

Reply:
I have documented my testing below https://github.com/e-mission/em-public-dashboard/pull/150#issuecomment-2359461028
In the `load_mongodump.sh`:
```
# Update the docker-compose configuration file with the actual DB_HOST
DB_HOST="mongodb://db/$DB_NAME"
sed -i.bak "s|DB_HOST:.*|DB_HOST: $DB_HOST|" "$CONFIG_FILE"
```
I understand, it would assign `mongdb://db$DB_NAME` to the configuration file.

I have also re-done testing and updated detailed logs below. https://github.com/e-mission/em-public-dashboard/pull/150#issuecomment-2365246025